src/main/scala/mesosphere/marathon/state/Group.scala

```diff
@@ -152,13 +153,58 @@ object Group extends StrictLogging {
   def validGroup(base: PathId, enabledFeatures: Set[String]): Validator[Group] =
     validator[Group] { group =>
       group.id is validPathWithBase(base)
-      group.apps.values as "apps" is every(
-        AppDefinition.validNestedAppDefinition(group.id.canonicalPath(base), enabledFeatures))
```

Comment:
Is there a reason that we did not validate the pods?

Reply:
do we know why is the pod validation missing here? It seems to me too that it should be here...

Reply:
I don't know why. I guess it was just missed.